recipe/meta.yaml

```
@@ -15,7 +17,10 @@ source:
 build:
   # Upstream does not build shared libs for Windows correctly
   skip: true  # [win]
-  number: 0
+  skip: true  # [cuda_compiler_version == "10.2"]
+  skip: true  # [cuda_compiler_version == "11.0"]
+  skip: true  # [cuda_compiler_version == "11.1"]
```

We could combine the skips like that
```suggestion
  skip: true  # [win or cuda_compiler_version in ("10.2", "11.0", "11.1")]
```